src/System.Data.Common/tests/System/Data/SqlTypes/SqlDecimalTest.cs

```diff
@@ -253,11 +253,11 @@ public void ArithmeticMethods()
         [Fact]
         public void AdjustScale()
         {
-            Assert.Equal("6464.646400", SqlDecimal.AdjustScale(_test1, 2, false).Value.ToString());
-            Assert.Equal("6464.65", SqlDecimal.AdjustScale(_test1, -2, true).Value.ToString());
-            Assert.Equal("6464.64", SqlDecimal.AdjustScale(_test1, -2, false).Value.ToString());
-            Assert.Equal("10000.000000000000", SqlDecimal.AdjustScale(_test2, 10, false).Value.ToString());
-            Assert.Equal("79228162514264337593543950335.00", SqlDecimal.AdjustScale(_test5, 2, false).ToString());
+            Assert.Equal(6464.646400.ToString(), SqlDecimal.AdjustScale(_test1, 2, false).Value.ToString());
```

Comment:
You should explicitly cast to the type (short or whatever) before calling To String, so you know you will get the same result.